modules/iam/main.tf

```diff
@@ -145,3 +146,50 @@ resource "aws_iam_role_policy" "restore_object_role_policy" {
   role   = aws_iam_role.restore_object_role.id
   policy = data.aws_iam_policy_document.restore_object_role_policy_document.json
 }
+
+
+# Step functions
+# copied from (cumulus/tf-modules/ingest/iam.tf) todo: Strip back and personalize for each step-function
```

Comment:
Cumulus was defining our IAM role for step-functions. Not ideal. This is the same over-permissive policy, but is more available for us to edit/limit to our needs.